recipes/configs/qwen2/0.5B_lora_single_device.yaml

```diff
@@ -19,11 +19,10 @@
 # Model Arguments
 model:
   _component_: torchtune.models.qwen2.lora_qwen2_0_5b
-  lora_attn_modules: ['q_proj', 'v_proj']
+  lora_attn_modules: ['q_proj', 'k_proj', 'v_proj']
```

Comment:
why did you add k_proj here?

Reply:
I had memory spare since increasing batch size OOM-ed, thought it might help performance